### ISSUE TITLE
TypeError: float() argument must be a string or a number, not 'Timeout'

### ISSUE THREAD

**Issue by kfxs** [coursera-dl/coursera-dl#611]
### Your environment
* Operating System (name/version): Ubuntu 16.04.2 LTS
* Python version: Python 3.5.2
* coursera-dl version: 0.9.0

### Subject of the issue
I get the following issue when executing  coursera-dl

### Actual behaviour

coursera_dl version 0.9.0
Downloading class: machine-learning (1 / 1)
Connection pool is full, discarding connection: www.coursera.org
Traceback (most recent call last):
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/util/timeout.py", line 124, in _validate_timeout
    float(value)
TypeError: float() argument must be a string or a number, not 'Timeout'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./coursera-dl", line 11, in <module>
    sys.exit(main())
  File "~/.local/lib/python3.5/site-packages/coursera/coursera_dl.py", line 237, in main
    error_occured, completed = download_class(args, class_name)
  File "~/.local/lib/python3.5/site-packages/coursera/coursera_dl.py", line 212, in download_class
    return download_on_demand_class(args, class_name)
  File "~/.local/lib/python3.5/site-packages/coursera/coursera_dl.py", line 125, in download_on_demand_class
    extractor = CourseraExtractor(session, args.username, args.password)
  File "~/.local/lib/python3.5/site-packages/coursera/extractors.py", line 31, in __init__
    login(session, username, password)
  File "~/.local/lib/python3.5/site-packages/coursera/cookies.py", line 145, in login
    headers=headers, allow_redirects=False)
  File "~/.local/lib/python3.5/site-packages/requests/sessions.py", line 560, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "~/.local/lib/python3.5/site-packages/requests/sessions.py", line 513, in request
    resp = self.send(prep, **send_kwargs)
  File "~/.local/lib/python3.5/site-packages/requests/sessions.py", line 623, in send
    r = adapter.send(request, **kwargs)
  File "~/.local/lib/python3.5/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py", line 565, in urlopen
    timeout_obj = self._get_timeout(timeout)
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py", line 308, in _get_timeout
    return Timeout.from_float(timeout)
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/util/timeout.py", line 154, in from_float
    return Timeout(read=timeout, connect=timeout)
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/util/timeout.py", line 97, in __init__
    self._connect = self._validate_timeout(connect, 'connect')
  File "~/.local/lib/python3.5/site-packages/requests/packages/urllib3/util/timeout.py", line 127, in _validate_timeout
    "int or float." % (name, value))
ValueError: Timeout value connect was Timeout(connect=None, read=None, total=None), but it must be an int or float.

**Comment by balta2ar** [comment]
Duplicate of https://github.com/coursera-dl/coursera-dl/issues/609#issuecomment-304868121

**Comment by kfxs** [comment]
$ pip3 show requests

Name: requests
Version: 2.17.3
Summary: Python HTTP for Humans.
Home-page: http://python-requests.org
Author: Kenneth Reitz
Author-email: me@kennethreitz.org
License: Apache 2.0
Location: ~/.local/lib/python3.5/site-packages
Requires: certifi, urllib3, chardet, idna

**Comment by kfxs** [comment]
downgrading to requests==2.10.0 fix the issue
